tests/test_app.py

```diff
@@ -26,6 +26,19 @@ def test_data_request():
                         "chat": {"id": "chat-1"},
                         "project": {"id": "PROJECT-KEY"},
                         "user": {"id": "", "title": ""},
+                        "token_usage": {
+                            "completion_tokens": 40,
+                            "prompt_tokens": 30,
+                            "cost": 0.001,
+                            "agg_cost": 0.001,
+                        },
+                        "source_deployment": "assistant",
+                        "dest_deployment": "gpt-4",
```

Comment:
I cannot see any usage of dest_deployment in the code. Is in correct?

Reply:
yes, it's correct. It seems we don't need this field.

Reply:
removed from core logs